libs/common/src/enums/feature-flag.enum.ts

```diff
@@ -11,5 +11,26 @@ export enum FeatureFlag {
   AC1795_UpdatedSubscriptionStatusSection = "AC-1795_updated-subscription-status-section",
 }
 
-// Replace this with a type safe lookup of the feature flag values in PM-2282
-export type FeatureFlagValue = number | string | boolean;
+type DefaultValue = boolean | number | string;
+
+/**
+ * Default value for feature flags.
+ *
+ * Flags MUST be short lived and SHALL be removed once enabled.
+ */
+export const DefaultFeatureFlagValue = {
+  [FeatureFlag.BrowserFilelessImport]: false as boolean,
+  [FeatureFlag.ItemShare]: false as boolean,
+  [FeatureFlag.FlexibleCollectionsV1]: false as boolean,
+  [FeatureFlag.VaultOnboarding]: false as boolean,
+  [FeatureFlag.GeneratorToolsModernization]: false as boolean,
+  [FeatureFlag.KeyRotationImprovements]: false as boolean,
+  [FeatureFlag.FlexibleCollectionsMigration]: false as boolean,
+  [FeatureFlag.ShowPaymentMethodWarningBanners]: false as boolean,
+  [FeatureFlag.EnableConsolidatedBilling]: false as boolean,
+  [FeatureFlag.AC1795_UpdatedSubscriptionStatusSection]: false as boolean,
+} satisfies Record<FeatureFlag, DefaultValue>;
```

Comment:
🎨 suggestion: Maybe it's just me but `as boolean` seems very distracting, could we do something like this?

```suggestion
const FALSE = false as boolean;

export const DefaultFeatureFlagValue = {
  [FeatureFlag.BrowserFilelessImport]: FALSE,
  [FeatureFlag.ItemShare]: FALSE,
  [FeatureFlag.FlexibleCollectionsV1]: FALSE,
  [FeatureFlag.VaultOnboarding]: FALSE,
  [FeatureFlag.GeneratorToolsModernization]: FALSE,
  [FeatureFlag.KeyRotationImprovements]: FALSE,
  [FeatureFlag.FlexibleCollectionsMigration]: FALSE,
  [FeatureFlag.ShowPaymentMethodWarningBanners]: FALSE,
  [FeatureFlag.EnableConsolidatedBilling]: FALSE,
  [FeatureFlag.AC1795_UpdatedSubscriptionStatusSection]: FALSE,
} satisfies Record<FeatureFlag, DefaultValue>;
```

Reply:
Agreed that's a nice helper.